Source/Extensions/Blazorise.DataGrid/_DataGridCellSelectEdit.razor.cs

```diff
@@ -67,10 +73,36 @@ private void OnSelectedValueChanged( object value )
         }
     }
 
+    protected override async Task OnAfterRenderAsync( bool firstRender )
+    {
+        if ( firstRender )
+        {
+            if ( ParentDataGrid.IsCellEdit && Column.CellEditing )
+            {
+                await Task.Yield();
```

Comment:
Didn't an issue with calling `Focus` without `await Task.Yield();` already fixed?

Reply:
Sorry, I don't follow?

Due to the nature of single threaded WASM this was needed from what I can remember

Reply:
I remember we that that workaround in the past. 

Here is the diff

![image](https://github.com/Megabit/Blazorise/assets/900302/9c7b34fc-8bf6-447d-ba9b-baf85ea9f4bd)

Reply:
Do you remember what led you to remove? Probably encountering same issue.
Anyway I can retest without it and I'll let you know. But I'm sure I'm not crazy and something was not working properly.

Reply:
It was fixed in .NET eventually, see https://github.com/dotnet/aspnetcore/pull/32017

Reply:
Just retried to remove it, does not work.